### PR TITLE
Updated UI of file uploader #7428

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by gautampanday** [comment]
s3-uploader

File: v1/upload.html [hunk 1]
```diff
@@ -1,36 +1,126 @@
...
+  <link href="css/bootstrap.min.css" rel='stylesheet'>
+  <script src="js/bootstrap.min.js" type="text/javascript"></script>
+  <link href="css/style.css" rel='stylesheet'>
+</head>
+<body>
+  <nav class="navbar navbar-default">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand navbar-brand-img" href="#">
+          <img alt="Brand" src="https://d1iqjce8nes3ts.cloudfront.net/static/img/openlabs-logo.png" class="brand-logo">
```
Comment (GauravButola) [inline v1/upload.html]: Pick image from local.

File: v1/upload.html [hunk 2]
```diff
@@ -1,36 +1,126 @@
...
+                <div class="tab-pane" id="profile">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?</div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
```
Comment (GauravButola) [inline v1/upload.html]: Why CDN when jquery is ther ein `src` folder

File: v1/upload.html [hunk 3]
```diff
@@ -1,36 +1,126 @@
...
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"></script>
```
Comment (GauravButola) [inline v1/upload.html]: remove.

File: v1/upload.html [hunk 4]
```diff
@@ -1,36 +1,126 @@
...
+    <script src="http://malsup.github.com/jquery.form.js"></script>
```
Comment (GauravButola) [inline v1/upload.html:54]: Pick from `src` folder.

File: v1/upload.html [hunk 5]
```diff
@@ -1,36 +1,126 @@
...
+          complete: function(xhr) {
+            status.html(xhr.responseText);
+          }
+        }); 
+
+      })       
     </script>
   </body>
-</html>
+  </html>
```
Comment (GauravButola) [inline v1/upload.html]: deindent.

File: v1/upload.html [hunk 6]
```diff
@@ -1,36 +1,126 @@
...
+
+        $('input:file').change(
+          function(){
+            if ($(this).val()) {
+              $('input:submit').attr('disabled',false);
+              $(".select-text").hide();
+              $( 'input:submit' ).removeClass("filter-scale" );
+            } 
+          }
+          );
```
Comment (GauravButola) [inline v1/upload.html]: deindent.

File: v1/upload.html [hunk 7]
```diff
@@ -1,36 +1,126 @@
...
+            value: parseData.fields[i].value,
+          }).appendTo('form');
+        }
+
+        $('<input>').attr({
+          type: 'file',
+          name: 'file'
+        }).appendTo('form');
+
+        $( 'input:file' ).addClass("btn btn-info" );
```
Comment (GauravButola) [inline v1/upload.html:74]: stray space.

File: v1/upload.html [hunk 8]
```diff
@@ -1,36 +1,126 @@
...
+    </section>
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"></script>
+    <script src="http://malsup.github.com/jquery.form.js"></script>
+    <script>
+      $(document).ready(function(){
+        search_string = window.location.search
+        parseData = JSON.parse(window.atob(search_string.substring(6, search_string.length)));
+
+        console.log(parseData.fields);
```
Comment (GauravButola) [inline v1/upload.html]: Remove

File: v1/upload.html [hunk 9]
```diff
@@ -1,36 +1,126 @@
...
+  <section class="section-spaces">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6">
+          <form action="" id="upload-form" method="POST" enctype="multipart/form-data" class="text-center">
+            <p class="select-text">Please select a file</p>
+            <br> 
+            <input type="submit" value="Upload File to S3" class="btn btn-info filter-scale" disabled>
+          </form>
+          <div class="progress">
```
Comment (GauravButola) [inline v1/upload.html]: Show only when upload starts.

**Comment by GauravButola** [comment]
LGTM

**Comment by PritishC** [comment]
The `<input type='file'>` line must be at the bottom of the dynamically placed fields. The S3 server is not able to find required form fields (such as `key`) otherwise.

File: v1/upload.html [hunk 10]
```diff
@@ -1,36 +1,133 @@
...
 
-			$('<input>').attr({
-				type: 'file',
-				name: 'file'
-			}).appendTo('form');
+        $('input:file').change(
+          function() {
+            if ($(this).val()) {
+              $(".select-file").hide();
+              $('input:submit').attr('disabled',false);
```
Comment (GauravButola) [inline v1/upload.html]: Put this in else block.

**Pull request merged**